article-list.html

```diff
@@ -37,49 +37,31 @@
                 <h1>All Articles</h1>
 
 				<!-- This is the category subheading that will change depending on which section of page user is on -->
-				<p class="subheading">Read articles about common skin issues</p>
+				<p class="subheading">Read articles about common skin questions</p>
 				<div class="image-first">
-					<a href="#"><img src="sample.png"></a>
+					<a href="fungal-acne.html"><img src="images/fungal_acne.jpg"></a>
 				</div>
 				<div class="title">
-					<a href="#">Article title will go here</a>
+					<a href="fungal-acne.html" class="first">What is Fungal Acne? (And how to treat it)</a>
+					<p>Learn what fungal acne is, how it's caused, and what you can do to treat it</p>

```

Comment:
Added a small caption underneath the article title so users briefly know what information the article covers.